src/assets/translations/en/main.json

```diff
@@ -351,6 +351,7 @@
     "rate": "Rate",
     "minerFee": "Miner Fee",
     "shapeshiftFee": "ShapeShift Fee",
+    "gasFeeExceedsTradeAmountThreshold": "Gas fee exceeds %{percentage}% of trade amount",
```

Comment:
I'm not sure if this one should be dynamic, to be confirmed with product. The way I understand it from the issue is it will always be a static 5. The dynamic amount is not very clear from a user perspective: if we're showing an amount, why do we have an "exceed" terminology?

Reply:
The "exceeds" is just to clarify to the user that the gas is over x% of their trade amount